velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMain.java

```diff
@@ -0,0 +1,58 @@
+package me.leoko.advancedban.velocity;
+
+import com.google.inject.Inject;
+import com.velocitypowered.api.event.proxy.ProxyInitializeEvent;
+import com.velocitypowered.api.event.Subscribe;
+import com.velocitypowered.api.event.proxy.ProxyShutdownEvent;
+import com.velocitypowered.api.plugin.Plugin;
+import com.velocitypowered.api.plugin.annotation.DataDirectory;
+import com.velocitypowered.api.proxy.ProxyServer;
+import me.leoko.advancedban.Universal;
+import me.leoko.advancedban.velocity.listener.ChatListenerVelocity;
+import me.leoko.advancedban.velocity.listener.ConnectionListenerVelocity;
+import org.slf4j.Logger;
+
+import java.nio.file.Path;
+
+@Plugin(
+    id = "advancedban",
+    name = "AdvancedBan",
+    version = "@version"
+)
+public class VelocityMain {
+
+  private final ProxyServer server;
+  private final Path dataDirectory;
+  private final Logger logger;
+
+  private static VelocityMain instance;
+
+  @Inject
+  public VelocityMain(ProxyServer server, @DataDirectory Path dataDirectory, Logger logger) {
+    this.instance = this;
```

Comment:
`instance` is not an instance field (ironically), so this assignment needs to reflect that